test/Libraries/DynamoPythonTests/IronPythonTests.csproj

```diff
@@ -46,18 +46,6 @@
       <HintPath>..\..\..\src\packages\IronPython.2.7.9\lib\net45\IronPython.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="IronPython.Modules, Version=2.7.9.0, Culture=neutral, PublicKeyToken=7f709c5b713576e1, processorArchitecture=MSIL">
```

Comment:
Removed these IronPython.Modules, IronPython.SQLLite and IronPython.Wpf binaries as they were not being used.